autotest.pm

```diff
@@ -32,6 +32,11 @@ OS Autoinst decides which test modules to run based on a distribution specific
 script called main.pm. This is either located in $vars{PRODUCTDIR} or
 $vars{CASEDIR} (e.g. <distribution>/products/<product>/main.pm).
 
+Wheels can be used to add functionality from other repositories. If a file
+wheels.yaml is present in $vars{PRODUCTDIR} (or $vars{CASEDIR} if unset) the
```

Comment:
It doesn't look like `PRODUCTDIR` is used anymore in the current state of the PR.